desc/equilibrium/equilibrium.py

```diff
@@ -623,6 +623,8 @@ def get_surface_at(self, rho=None, theta=None, zeta=None):
             AR = np.zeros((surface.R_basis.num_modes, self.R_basis.num_modes))
             AZ = np.zeros((surface.Z_basis.num_modes, self.Z_basis.num_modes))
 
+            # TODO: Don't call zernike_radial for each mode, instead compute all
```

Comment:
is there an issue for this so we don't lose track of it? and for this in the linear objectives too